slicops/package_data/static/layout/screen.yaml

```diff
@@ -0,0 +1,21 @@
+columns:
+- layout: sm-3
+  rows:
+  - name: beam_path
+  - name: camera
+  - name: pv
+  - name: camera_gain
+  - row:
```

Comment:
Perhaps `row` should be called `cell`. We could define a cell to layout horizontally. It could also be called "group" but there are already lots of groups.

Reply:
I've changed it to CellComponent with selector widget-cell